manila/share/drivers/netapp/dataontap/protocols/nfs_cmode.py

```diff
@@ -21,7 +21,8 @@
 import six
 
 from manila.common import constants
-from manila import exception
+from manila import exception, utils
```

Comment:
Please fix the pep8 error `H301: one import per line`

Reply:
sure, it's fixed now

Reply:
Now you removed the import completely, no?
It is still needed, it should just go to a separate line ;)